concourse/scripts/ssh.sh

```diff
@@ -1,16 +1,45 @@
 #!/bin/bash
 
 set -euo pipefail
-key=/tmp/concourse_id_rsa.$RANDOM
-trap 'rm -f $key' EXIT
+TUNNEL=${1:-}
+SOCKET=~/ssh_tunnel_socket.ctl
```

Comment:
Can this socket be put somewhere other then our homedir.  Maybe under `$TMPDIR`, or somewhere within the paas-cf directory tree.